### ISSUE TITLE
Add `fallbackLocale` to nuxt.config options

### ISSUE THREAD

**Issue by warflash** [nuxt-modules/i18n#2753]
### Describe the feature

Figuring out how to set the `fallbackLocale` seems to be a quite common question and point of confusion (happened to myself just now and checked on the nuxt discord). It feels very natural to want to stick that property into `nuxt.config` and not create a seperate `i18n.config.ts`.

I'd like to propose the addition of that setting to the possible settings to the i18n object in `nuxt.config`. (If this has been asked and denied earlier then I apologize, couldn't find an issue though).
We're dealing with ~10 layer apps that extend off of a shared nuxt layer in a monorepo and adding 10 i18n.config files feels like an suboptimal solution for something that could also be achieved with a single line of code.

Thanks!

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution) (The contribution guideline of nuxt-modules/i18n is compliant with Nuxt too).
- [X] Check existing [discussions](https://github.com/nuxt-modules/i18n/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by BobbieGoede** [comment]
You're right, the configuration split between `vue-i18n` options and module specific options have clearly been a point of confusion.

I'm open to suggestions on how to improve/clarify this while taking into account [the reasoning](https://github.com/nuxt-modules/i18n/pull/1948#issuecomment-1482749302) behind moving `vue-i18n` options to a separate file. I think the main issue was that `vue-i18n` configuration can contain non-serializable values such as functions, which we worked around using code generation.

**Comment by silvio-e** [comment]
I was also struggling with this and needed quite some time to figure it out. I expected the config to work like it does for many other modules I have worked with. Configuration either in the `nuxt.config` or in an extra config file. But here it happens in two places.

Before any code changes I suggest to update the documentation and clarify following points:

1. The difference/relation between nuxt/i18n & vue/i18n. If one hasn't worked with them it is confusing.
2. Highlight what is nuxt/i18n config and what is vue/i18n config

I think this would help already very much.

**Comment by MickL** [comment]
You guys need to keep in mind that [nuxt-i18n](https://i18n.nuxtjs.org) is a wrapper around [vue-i18n](https://vue-i18n.intlify.dev). Therefor it makes sense that nuxt.config is for configuring nuxt-i18n and another file is needed to call [createI18n()](https://vue-i18n.intlify.dev/guide/essentials/started.html#creating-the-i18n-instance) at **runtime**.
